docs/en/administration/resource_group.md

```diff
@@ -70,12 +70,13 @@ You can specify CPU and memory resource quotas for a resource group on a BE by u
 
   The CPU core limit for this resource group on a single BE node. It takes effect only when it is set to greater than `0`. Range: [0, `avg_be_cpu_cores`], where `avg_be_cpu_cores` represents the average number of CPU cores across all BE nodes. Default: 0.
 
-- `spill_mem_limit_threshold`:
+- `spill_mem_limit_threshold`
 
-  The memory usage threshold (percentage) at which a resource group triggers the spilling of intermediate results. The valid range is [0, 1], and the default value is 1. This parameter was introduced in v3.1.7.
-
-  - If spilling is enabled for a query and the query hits a resource group (including the default resource group `default_wg`), when the memory used by the query exceeds `query_mem_limit * mem_limit * spill_mem_limit_threshold`, intermediate results are spilled to disk to reduce memory usage. Here, `query_mem_limit` indicates the memory limit for a query on a BE node, controlled by the system variable `query_mem_limit`.
-  - If spilling is enabled for a query but the query is not managed by the resource group feature, the system will decide whether to trigger spilling based on query's actual memory usage.
+  The memory usage threshold (percentage) at which a resource group triggers the spilling of intermediate results. The valid range is (0, 1). The default value is 1, indicating the threshold does not take effect. This parameter was introduced in v3.1.7.
+  - If automatic spilling is enabled (that is, the system variable `spill_mode` is set to `auto`) but the resource group feature is disabled, the system will trigger spilling when the memory usage of a query exceeds 80% of `query_mem_limit`. Here, `query_mem_limit` is the maximum memory that a single query can use, controlled by the system variable `query_mem_limit`, with a default value of 0, indicating no limit.
+  - If automatic spilling is enabled, and the query hits a resource group (including all system built-in resource groups), spilling will be triggered if the query meets any of the following:
+    - when the memory used by all queries in the current resource group exceeds `current BE node memory limit * mem_limit * spill_mem_limit_threshold`
```

Comment:
delete "when", use a complete sentence "The memory used by all queries in the current resource group exceeds xxx."